src/Components/Forms/src/EditContext.cs

```diff
@@ -66,6 +66,11 @@ public FieldIdentifier Field(string fieldName)
     /// </summary>
     public EditContextProperties Properties { get; }
 
+    /// <summary>
+    /// Gets whether field identifiers should be generated for &lt;input&gt; elements.
+    /// </summary>
+    public bool ShouldUseFieldIdentifiers => !OperatingSystem.IsBrowser();
```

Comment:
Can you make this a default value instead of a lambda, so that it can be turned on explicitly if desired.